File: Angular/Angular/src/Model/Donor.ts
import { EmailValidator } from "../../node_modules/@angular/forms/index";
import { Gift } from "./Gift";

export class Donor {
    public id: number= 0;
    public firstName: string = '';
    public lastName: string = '';
    public gifts: Gift[] = [];
    public email:  string = '';
    public phone: string = '';
  
}

File: WebApi/WebApi/Models/Donor.cs
namespace WebApi.Models
{
    public class Donor
    {
        public int Id { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public List<Gift> Gifts { get; set; }=new List<Gift>();
        public string Email { get; set; }=string.Empty;
        public string Phone { get; set; }= string.Empty;
    }
}
